docs/firstobject.md

```diff
@@ -14,42 +15,41 @@ Note: This guide will assume Archipelago is running on `http://localhost:8001`,
 
 ## Welcome!
 
-Start by opening `http://locahost:8001` in your favourite Web Browser.
+Start by opening `http://locahost:8001` in your favorite Web Browser.
 
-![Welcome](../imgs/Welcome.jpg)
+![Welcome](../imgs/01_ingest_log-in_beta2.png)
 
 Your Demo deployment will have a fancy Home page with some banners and a small explanation of what Archipelago is and can do. Feel free to read through that now or later.
 
-Click on `Log in` and use your `demo` credentials from them deployment guide.
+Click on `Log in` in the top left corner and use your `demo` credentials from the deployment guide.
 
-- User: demo
-- pass: demo 
+- user: demo
+- pass: demo
 
 (or whatever password you decided was easy for you to remember during the deployment phase)
 
-![Log In](../imgs/Log-in.jpg)
-
 Press the `Log in` button.
 
-![Logged In](../imgs/Logged-in.jpg)
+![Log In](../imgs/02_ingest_demo-demo_beta2_cropped.png)
```

Comment:
Ok, this applies to every screenshot: too big, in image size and filesize. Which leads to onfocused images in the actual viewing. Make them JPEG and scale them to the actual image size please, assume a screen of max 1280 pixels width (or less).